docs/1_overview/cybersecurity/malware.md

```
@@ -0,0 +1,27 @@
+- Virus
+- Logic bombs
+- Wipers
+- Key loggers
+- Trojan
+- Spywares
+- Adware
+- Botnets
+- Rootkit
+- [[cryptojacking|Cryptojacking]]
+
+### Further resources
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)